File: front/src/entity/record/record-date.ts
export class RecordDate {
  private publicationDate: string;
  private manufactureDate: string;

  getPublicationDate(): string {
    return this.publicationDate;
  }

  setPublicationDate(value: string) {
    this.publicationDate = value;
  }

  getManufactureDate(): string {
    return this.manufactureDate;
  }

  setManufactureDate(value: string) {
    this.manufactureDate = value;
  }
}

File: back/src/main/java/org/lenndi/umtapo/solr/document/bean/record/RecordDate.java
package org.lenndi.umtapo.solr.document.bean.record;

import com.fasterxml.jackson.annotation.JsonInclude;

/**
 * Record date entity.
 */
@JsonInclude(JsonInclude.Include.NON_NULL)
public class RecordDate {

    private String publicationDate;
    private String manufactureDate;

    /**
     * Gets publication date.
     *
     * @return the publication date
     */
    public String getPublicationDate() {
        return publicationDate;
    }

    /**
     * Sets publication date.
     *
     * @param publicationDate the publication date
     */
    public void setPublicationDate(String publicationDate) {
        this.publicationDate = publicationDate;
    }

    /**
     * Gets manufacture date.
     *
     * @return the manufacture date
     */
    public String getManufactureDate() {
        return manufactureDate;
    }

    /**
     * Sets manufacture date.
     *
     * @param manufactureDate the manufacture date
     */
    public void setManufactureDate(String manufactureDate) {
        this.manufactureDate = manufactureDate;
    }
}
